### PR TITLE
Added docker-cloud exporter

### PR DESCRIPTION
**Pull request by Rucknar**
Added new exporter, early days but it's functional and i'm expecting the volume of metrics to increase over the coming weeks.

### CODE REVIEW

File: content/docs/instrumenting/exporters.md [hunk 1]
```diff
@@ -72,6 +72,7 @@ wide variety of JVM-based applications, for example [Kafka](http://kafka.apache.
    * [AWS ECS exporter](https://github.com/slok/ecs-exporter)
    * [Cloudflare exporter](https://github.com/wehkamp/docker-prometheus-cloudflare-exporter)
    * [Docker Hub exporter](https://github.com/infinityworksltd/docker-hub-exporter)
+   * [Docker Cloud exporter](https://github.com/infinityworksltd/docker-cloud-exporter)
```
Comment (brian-brazil) [inline content/docs/instrumenting/exporters.md]: This sorts one up

**Comment by Rucknar** [comment]
@brian-brazil Any docs/guides? Not come across ConstMetrics before.

**Comment by Rucknar** [comment]
@brian-brazil  Fixed sorting

**Comment by brian-brazil** [comment]
https://github.com/prometheus/consul_exporter/blob/master/consul_exporter.go is one example

**Pull request merged**